### PR TITLE
[WIP] Add error messages in new windows

### PR DESCRIPTION
**Pull request by jchyzy**

### CODE REVIEW

File: gui/functionality/idact_notebook.py [hunk 1]
```diff
@@ -114,6 +114,8 @@ def deploy_notebook(self):
                 notebook.open_in_browser()
                 sleep_until_allocation_ends(nodes=nodes)
         except:  # noqa, pylint: disable=broad-except
+            self.window = ErrorApp()
+            self.window.show()
             if log is not None:
```
Comment (PiekarzKarolina) [inline gui/functionality/idact_notebook.py]: You can delete this if about log and the log itself. It is defined as none and never changed. And now we have something to put in the body of except so it isn't needed.

File: gui/functionality/remove_cluster.py [hunk 2]
```diff
@@ -16,6 +16,10 @@ def remove_cluster(self):
         cluster_name = self.ui.cluster_name_removec_edit.text()
         self.parameters['remove_cluster_arguments']['cluster_name'] = cluster_name
         self.saver.save(self.parameters)
-        remove_cluster(cluster_name)
+        try:
+            remove_cluster(cluster_name)
+        except KeyError as e:
+            self.window = ErrorApp("Cluster not exists")
```
Comment (PiekarzKarolina) [inline gui/functionality/remove_cluster.py]: Shouldn't it be better with text `Cluster does not exist`?

File: gui/idact_app.py [hunk 3]
```diff
@@ -8,6 +8,7 @@
 Ui_AddNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/add-native.ui')
 Ui_RemoveNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/remove-native.ui')
 Ui_ShowNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/show-native.ui')
+Ui_Error, QtBaseClass = uic.loadUiType('../widgets_templates/error.ui')
```
Comment (PiekarzKarolina) [inline gui/idact_app.py]: Could you change all `QtBaseClass` into `_QtBaseClass`? I think it is the convention of naming not used variables.

File: gui/functionality/idact_notebook.py [hunk 4]
```diff
@@ -114,6 +114,8 @@ def deploy_notebook(self):
                 notebook.open_in_browser()
                 sleep_until_allocation_ends(nodes=nodes)
         except:  # noqa, pylint: disable=broad-except
+            self.window = ErrorApp()
```
Comment (PiekarzKarolina) [inline gui/functionality/idact_notebook.py]: Maybe `e` has some sort of a title that can be passed as an error message?

File: gui/functionality/idact_notebook.py (hunk 1 above)
Comment (jchyzy) [reply gui/functionality/idact_notebook.py]: Ok, I deleted it.

File: gui/functionality/remove_cluster.py (hunk 2 above)
Comment (jchyzy) [reply gui/functionality/remove_cluster.py]: Ok, changed.

File: gui/idact_app.py (hunk 3 above)
Comment (jchyzy) [reply gui/idact_app.py]: Done.

File: gui/functionality/idact_notebook.py (hunk 4 above)
Comment (jchyzy) [reply gui/functionality/idact_notebook.py]: I changed message to "An error occured while deploing notebook".

**Pull request merged**